src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/reducers/__tests__/chartBuilderReducer.test.ts

```diff
@@ -100,13 +103,25 @@ describe('chartBuilderReducer', () => {
         payload: testChartDefinition,
       };
 
-      const nextState = produce(chartBuilderReducer)(initialState, action);
+      const nextState = produce(
+        chartBuilderReducer({
+          data: [],
+          meta: testFullTable.subjectMeta,
+          tableTitle: undefined,
```

Comment:
Similar to an earlier comment about a similar property - we don't need to set `tableTitle` to `undefined` as it's optional.

There's a bunch of other places throughout this test suite we can remove this from as well.